pkg/controller/networkaddonsconfig/networkaddonsconfig_controller_test.go

```diff
@@ -0,0 +1,123 @@
+package networkaddonsconfig
+
+import (
+	"fmt"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/names"
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type checkUnit struct {
+	key           string
+	shouldExist   bool
+	expectedValue string
+}
+
+var _ = Describe("Networkaddonsconfig", func() {
+	Context("When CR is not labeled", func() {
+		It("Should find only default labels", func() {
+			renderData := render.MakeRenderData()
```

Comment:
MakeRenderData+RenderTemplate should be in BeforeEach() IMO, as it is more related to `Context("When CR is not labeled"` and not to the `It("Should find only default labels"` part